README.md

```diff
@@ -102,240 +91,7 @@ report results.
-A short list of modules will be already available for you. They are all the imports of
-[this file](https://github.com/scanapi/scanapi/blob/master/scanapi/evaluators/code_evaluator.py#L1).
-
-### Chaining Requests
-
-Inside the request scope, you can save the results of the response to use in the next
-requests. For example:
-
-```yaml
-requests:
-  - name: list_all
-    method: get
-    vars:
-      dev_id: ${{response.json()[2]["uuid"]}}
-```
-
-The dev_id variable will receive the `uuid` value of the 3rd result from the devs_list_all request
-
-If the response is
-
-```json
-[
-    {
-        "uuid": "68af402f-1084-40a4-b9b2-6bb5c2d11559",
-        "name": "Anna",
-        "yearsOfExperience": 5,
-        "languages": [
-            "python",
-            "c"
-        ],
-        "newOpportunities": true
-    },
-    {
-        "uuid": "0d1bd106-c585-4d6b-b3a4-d72dedf7190e",
-        "name": "Louis",
-        "yearsOfExperience": 3,
-        "languages": [
-            "java"
-        ],
-        "newOpportunities": true
-    },
-    {
-        "uuid": "129e8cb2-d19c-41ad-9921-cea329bed7f0",
-        "name": "Marcus",
-        "yearsOfExperience": 4,
-        "languages": [
-            "c"
-        ],
-        "newOpportunities": false
-    }
-]
-```
-
-The dev_id variable will receive the value `129e8cb2-d19c-41ad-9921-cea329bed7f0`
-
-### API specification in multiple files
-
-With `!include`, it is possible to build your API specification in multiple files.
-
-For example, these two files
-
-```yaml
-# scanapi.yaml
-endpoints:
-  - name: scanapi-demo
-    path: ${BASE_URL}
-    requests: !include include.yaml
-```
-
-```yaml
-# include.yaml
-- name: health
-  path: /health/
-```
-
-would generate:
-
-```yaml
-endpoints:
-  - name: scanapi-demo
-    path: ${BASE_URL}
-    requests:
-      - name: health
-        path: /health/
-```
-
-### Configuration File
-
-If you want to configure the ScanAPI with a file, you can create a `.scanapi.yaml` file in the root
-of your project
-
-```yaml
-project_name: DemoAPI # This will be rendered in the Report Title.
-spec_path: my_path/scanapi.yaml # API specification file path
-output_path: my_path/my-report.html # Report output path.
-template: my_template.jinja # Custom report template path.
-```
-
-### Hiding sensitive information
-
-If you want to omit sensitive information in the report, you can configure it in the `.scanapi.yaml`
-file.
-
-```yaml
-report:
-  hide-request:
-    headers:
-      - Authorization
-```
-
-The following configuration will print all the headers values for the `Authorization` key for all
-the request as `SENSITIVE_INFORMATION` in the report.
-
-In the same way you can omit sensitive information from response.
-
-```yaml
-report:
-  hide-response:
-    headers:
-      - Authorization
-```
-
-Available attributes to hide: `headers`, `body` and `url`.
+The full documentation is available at [scanapi.dev][https://scanapi.dev]!
```

Comment:
The hyperlink should be enclosed in parenthesis here :)
Ex:
`The full documentation is available at [scanapi.dev](https://scanapi.dev)!`